Library/Homebrew/cmd/list.rb

```diff
@@ -40,6 +40,12 @@ def list_args
       switch "--pinned",
              description: "List only pinned formulae, or only the specified (pinned) "\
                           "formulae if <formula> are provided. See also `pin`, `unpin`."
+      switch "--requested",
+             depends_on:  "--formula",
+             description: "List only installed formulae that were explicitly asked for."
```

Comment:
```suggestion
             description: "List only installed formulae that were installed on request."
```
Let's keep the language consistent with use elsewhere so that users don't have to guess about whether they mean the same thing.